docs/6.x/pack.md

```diff
@@ -912,3 +912,38 @@ When building a Cluster Image, `tele build` will discover `custom-planet:1.0.0`
 Docker image and vendor it along with other dependencies. During the Cluster
 installation all nodes with the role `worker` will use the custom "planet" Docker
 image instead of the default one.
+
+## Direct Upgrades From Older LTS Versions
+
+Kubernetes has a strict version skew [support policy](https://kubernetes.io/docs/setup/release/version-skew-policy/#supported-version-skew) which makes
+cluster upgrades to versions of Kubernetes being more than a couple of minor versions apart complicated if not impossible, hence
```

Comment:
Sorry, I think this conceptually could be introduced a bit cleaner. I'm not a huge fan of the part about kubernetes upgrades being complicated / impossible, I think that portion doesn't really add value to the docs.

I would consider starting by introducing the gravity requirement on upgrades between adjacent LTS releases (or link to where we document this, but I don't think we do), and then that can indicate its partially driven by the kubernetes version skew policy, but also that's the upgrade paths we officially test. Then I wouldn't bother with the content about what a "correct" upgrade process is, as this is a feature that can be used to bypass those requirements.



Also, questions I can envision being asked I don't think are covered:
- Can this be used with any cluster, or only with "old" clusters that need the intermediate step.
- Must the --upgrade-via be an LTS version?
- Will the upgrade take longer?
- Will the image be larger?

Reply:
Thanks @knisbet - I'll update!